src/main/java/com/extendedclip/deluxemenus/utils/Constants.java

```diff
@@ -42,6 +42,8 @@ private Constants() {
     public static final String ITEMSADDER_PREFIX = "itemsadder-";
     public static final String ORAXEN_PREFIX = "oraxen-";
     public static final String MMOITEMS_PREFIX = "mmoitems-";
+    public static final String EXECUTABLEITEMS_PREFIX = "ei-";

```

Comment:
Please use the full name for the prefix, `executableitems-` and `executableblocks-`.